src/utils/axios.js

```diff
@@ -1,10 +1,11 @@
 import axios from 'axios';
 import localstorage from './localstorage';
 
-export const Axios = () => {
+export const Axios = (token = null) => {
   return axios.create({
     headers: {
       'Content-Type': 'application/json',
+      Authorization: token,
```

Comment:
Would you consider getting the token and setting it to to the Authorization header instead of passing it as a parameter? i.e `Authorization: localstorage.get().token || null`